src/main/resources/db/changelog/changes/01-create-books-table.yaml

```diff
@@ -40,4 +40,10 @@ databaseChangeLog:
               - column:
                   name: cover_image
                   type: varchar(255)
+              - column:
+                  name: is_deleted
+                  type: bit
+                  defaultValue: 'false'
+                  constraints:
+                    nullable: false

```

Comment:
does Liquibase allow to modify existing migrations? O_o

double check whether everything works properly after this update

the previous approach with separate changeLog for a new column was fine, there was no need to remove it :)